Sources/Navidux/Core/Navigation/NavigationController/DefaultNavigationController.swift

```diff
@@ -1,6 +1,6 @@
 import UIKit
 
-public final class NavigationControllerImpl: UINavigationController, NavigationController {
+public final class DefaultNavigationController: UINavigationController, NavigationController {
```

Comment:
it's not defalut at all😅 naming need to be fixed

Reply:
Default prefix does not mean that it is standard `UINavigationController` provided by `UIKit` framework, it means that this entity is defaulted as `NavigationController` in the library. Pretty common practice in naming this kind of entities in the libraries

Reply:
i think `Base` is better word for prefix, cause in this implementation there are a lot of logic related to library. and you what to force user of library inherit `BaseNavigationController`

Reply:
Aight, will change the naming to use `Base` prefix